app/Controller/SongsController.php

```diff
@@ -244,32 +244,21 @@ public function albums() {
                 )
             );
         } else {
-            $subQuery = $db->buildStatement(
-                array(
-                    'fields' => array('MIN(subsong.id)', 'subsong.album'),
-                    'table' => $db->fullTableName($this->Song),
-                    'alias' => 'subsong',
-                    'group' => 'subsong.album'
-                ),
-                $this->Song
-            );
-            $subQuery = ' (Song.id, Song.album) IN (' . $subQuery . ') ';
-
             if ($page == 1) {
                 $latests = $this->Song->find('all', array(
-                    'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
-                    'conditions' => $subQuery,
+                    'fields' => array('MIN(Song.id)', 'Song.band', 'Song.album', 'Song.cover'),
                     'order' => 'Song.created DESC',
+                    'group' => 'Song.album',
```

Comment:
Create a new branch (based on `development` or your actual branch, depending on the base code you depend on), cherry-pick gs11@af91a03 into this branch. Make a PR from this branch to dev.

In this actual branch, I guess `git rebase -i 57d1531` and there you remove commit af91a03. Force push, and that should do it.